modules/Currencies/Currency.php

```diff
@@ -497,7 +497,7 @@ function format_money($amount, $for_display = true)
  *(default ".").  Special case: when num_grp_sep is ".", it will return NULL as the num_grp_sep.
  * @return array Two element array, first item is num_grp_sep, 2nd item is dec_sep
  */
-function get_number_seperators($reset_sep = false)
+function get_number_separators($reset_sep = false)
```

Comment:
I think what we'd need to do here is have another function above this with the original name like this:

```php
/**
 * @deprecated 
 * @param bool $reset_sep
 */
function get_number_seperators($reset_sep = false)
{
    LoggerManager::getLogger()->deprecated('get_number_seperators will be removed in a future release, please
    update your code to use get_number_separators');
    get_number_separators($reset_sep);
}
```

That would allow us to keep backwards compatibility for those that will still be using the old function.